mypy/fastparse.py

```diff
@@ -769,6 +769,22 @@ def visit_Str(self, n: ast35.Str) -> Union[UnicodeExpr, StrExpr]:
         else:
             return UnicodeExpr(n.s)
 
+    # Only available with typed_ast >= 0.6.2
+    if hasattr(ast35, 'JoinedStr'):
+        # JoinedStr(expr* values)
+        @with_line
+        def visit_JoinedStr(self, n: ast35.JoinedStr) -> StrExpr:
```

Comment:
You're also going to have to add `JoinedStr` and `FormattedValue` to the stubs for `typed_ast.ast35` in [typeshed](https://github.com/python/typeshed).